modules/sizeMappingV2.js

```diff
@@ -0,0 +1,464 @@
+        logInfo(`SizeMappingV2:: AdUnit: ${adUnit.code}, Bidder: ${bidderCode} - Transformed mediaTypes after filtration: `, transformedMediaTypes);
+        logInfo(`SizeMappingV2:: AdUnit: ${adUnit.code}, Bidder: ${bidderCode} - mediaTypes that got filtered out: `, Object.keys(mediaTypes).filter(mt => Object.keys(transformedMediaTypes).indexOf(mt) === -1));
+
+        // check if adUnit has any active media types remaining, if not drop the adUnit from auction,
+        // else proceed to evaluate the bids object.
+        if (Object.keys(transformedMediaTypes).length === 0) {
+          logInfo(`SizeMappingV2:: Ad Unit: ${adUnit.code} is disabled since there are no active media types after sizeConfig filtration.`);
+          return result;
+        }
+        result
+          .push(adUnit.bids.filter(bid => bid.bidder === bidderCode)
+            .reduce((bids, bid) => {
+              if (isLabelActivated(bid, labels, adUnit.code)) {
+                // handle native params
+                const nativeParams = adUnit.nativeParams || deepAccess(adUnit, 'mediaTypes.native');
+                if (nativeParams) {
+                  bid = Object.assign({}, bid, {
+                    nativeParams: processNativeAdUnitParams(nativeParams)
+                  });
+                }
+
+                bid = Object.assign({}, bid, getDefinedParams(adUnit, ['mediaType', 'renderer']));
+
+                if (bid.sizeConfig) {
+                  const relevantMediaTypes = getRelevantMediaTypesForBidder(bid.sizeConfig, activeViewport);
+                  if (relevantMediaTypes.length === 0) {
+                    logError(`SizeMappingV2:: Bidder: ${bid.bidder} in Ad Unit: ${adUnit.code} has not configured sizeConfig property correctly. This bidder won't be eligible for sizeConfig checks and will remain active.`);
+                    bid = Object.assign({}, bid);
+                  } else if (relevantMediaTypes[0] !== 'none') {
+                    const bidderMediaTypes = Object
+                      .keys(transformedMediaTypes)
+                      .filter(mt => relevantMediaTypes.indexOf(mt) > -1)
+                      .reduce((mediaTypes, mediaType) => {
+                        mediaTypes[mediaType] = transformedMediaTypes[mediaType];
+                        return mediaTypes;
+                      }, {});
+
+                    if (Object.keys(bidderMediaTypes).length > 0) {
+                      bid = Object.assign({}, bid, { mediaTypes: bidderMediaTypes });
+                    } else {
+                      logInfo(`SizeMappingV2:: Bidder: ${bid.bidder} in Ad Unit: ${adUnit.code} is disabled.`);
+                      return bids;
+                    }
+                  } else {
+                    logInfo(`SizeMappingV2:: Bidder: ${bid.bidder} in Ad Unit: ${adUnit.code} is disabled due to failing sizeConfig check.`);
```

Comment:
This message should only show when the bidder has the relevantMediaTypes field set to none for this size tier correct?  If so, can we update the message to be more reflecting of that condition?  

The current message feels like they did something wrong in their setup, rather than it be the result of a choice they made.

Reply:
Made the change.